egs/librispeech/s5/local/chain/tuning/run_tdnn_lstm_1n.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+# this is the tdnn-lstmp based on the run_tdnn_lstm_1n.sh under Switchboard.
+# local/chain/compare_wer_rnnlm.sh exp/chain_cleaned/tdnn_lstm1n_rnnlm_sp/
```

Comment:
Can you please make clear what commands would be necessary to reproduce this result?
Because the RNNLM rescoring commands don't appear here.  And please make the affix in the comment match the affix in the script itself (I don't see an _rnnlm).

Reply:
Resolved. I'm decoding other setups. Hope to get better results than this one.